frontend/micro-ui/web/micro-ui-internals/packages/modules/Project/src/pages/employee/CreateProject/CreateProjectResponse.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from "react";
```

Comment:
@hariomsinha-eGov Can we make this component as a generic comp in react-components library. We have multiple response comps present in several modules. We have one generic one in works/src/components/response.js

Reply:
yes, we can have that. Also, in the projects case, the banner had the updates to show multiple IDs in diff rows. That change is done in Banner ( one from react components )